README.md

```diff
@@ -62,46 +44,43 @@ Notes on functionality:
 * Peeps (posts to chitter) have the name of the maker and their user handle.
 * Your README should indicate the technologies used, and give instructions on how to install and run the tests.
 
-Bonus:
------
-
-If you have time you can implement the following:
+## How to use
 
-* In order to start a conversation as a maker I want to reply to a peep from another maker.
+### To set up the project
 
-And/Or:
+Clone this repository and then run:
 
-* Work on the CSS to make it look good.
+```
+bundle
+```
 
-Good luck and let the chitter begin!
+### To set up the database
 
-Code Review
------------
+Connect to `psql` and create the `chitter` and `chitter_test` databases:
 
-In code review we'll be hoping to see:
+```
+CREATE DATABASE chitter;
+CREATE DATABASE chitter_test;
+```
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+To set up the appropriate tables, connect to each database in `psql` and run the SQL scripts in the `db/migrations` folder in the given order.
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
+### To run the Chitter app:
 
-Notes on test coverage
-----------------------
+```
+rackup -p 9292
+```
```

Comment:
I think the default port is 9292 so you can just run rackup - unless you want to run it on a non default port